sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ImplementIDisposableCorrectly.cs

```diff
@@ -174,12 +174,13 @@ private void VerifyDisposeOverride(MethodDeclarationSyntax disposeMethod)
 
             private void VerifyDispose(MethodDeclarationSyntax disposeMethod, bool isSealedClass)
             {
-                if (disposeMethod?.Body == null)

```

Comment:
because of this change, ITs are failing with

> CSC : error AD0001: Analyzer 'SonarAnalyzer.Rules.CSharp.ImplementIDisposableCorrectly' threw an exception of type 'System.ArgumentException' with message 'Syntax node is not within syntax tree'. [C:\Workspace\sonar-csharp\sonaranalyzer-dotnet\its\sources\akka.net\src\core\Akka\Akka.csproj]